context/test_notebooks.py

```diff
@@ -43,7 +44,7 @@ def get_notebooks(directory: str) -> Iterable[str]:
                     any(match_ignored_file in file for match_ignored_file in ignored_filenames)
                 )
                 and "MNMG" not in file.upper()
-                and file not in ignored_notebooks
+                and not (any(file in ignored_notebook for ignored_notebook in ignored_notebooks))
```

Comment:
Typically Python isn’t written with parentheses around the not operator.
```suggestion
                and not any(file in ignored_notebook for ignored_notebook in ignored_notebooks)
```